packages/atomic/src/components/common/generated-answer/render-answer-content.spec.ts

```diff
@@ -0,0 +1,311 @@
+          citations: [{id: '1'}, {id: '2'}],
+        },
+      });
+
+      expect(renderSourceCitations).toHaveBeenCalledWith({
+        props: expect.objectContaining({
+          label: 'Citations',
+          isVisible: true,
+        }),
+      });
+    });
+
+    it('should call renderDisclaimer with correct arguments when not streaming', async () => {
+      await renderComponent({
+        isAnswerVisible: true,
+        hasRetryableError: false,
+        // @ts-expect-error Test fixture with partial mock
+        generatedAnswerState: {
+          isStreaming: false,
+        },
+      });
+
+      expect(renderDisclaimer).toHaveBeenCalledWith({
+        props: expect.objectContaining({
+          i18n,
+          isStreaming: false,
+        }),
+      });
+    });
+
+    it('should call renderGeneratingAnswerLabel with correct arguments', async () => {
+      await renderComponent({
+        isAnswerVisible: true,
+        hasRetryableError: false,
+        collapsible: true,
+        // @ts-expect-error Test fixture with partial mock
+        generatedAnswerState: {
+          isStreaming: true,
+        },
+      });
+
+      expect(renderGeneratingAnswerLabel).toHaveBeenCalledWith({
+        props: expect.objectContaining({
+          i18n,
+          isStreaming: true,
+          collapsible: true,
+        }),
+      });
+    });
+
+    describe('when collapsible is true and not streaming', () => {
+      it('should call renderShowButton when expanded is true', async () => {
+        const onClickShowButton = vi.fn();
+
+        await renderComponent({
+          collapsible: true,
+          onClickShowButton,
+          // @ts-expect-error Test fixture with partial mock
+          generatedAnswerState: {
+            isStreaming: false,
+            expanded: true,
+          },
+        });
+
+        expect(renderShowButton).toHaveBeenCalledWith({
+          props: expect.objectContaining({
+            i18n,
+            onClick: onClickShowButton,
+            isCollapsed: false,
+          }),
+        });
+      });
+
+      it('should call renderShowButton when expanded is false', async () => {
+        await renderComponent({
+          collapsible: true,
+          // @ts-expect-error Test fixture with partial mock
+          generatedAnswerState: {
+            isStreaming: false,
+            expanded: false,
+          },
+        });
+
+        expect(renderShowButton).toHaveBeenCalledWith({
+          props: expect.objectContaining({
+            isCollapsed: true,
+          }),
+        });
+      });
+
+      it('should not call renderShowButton when streaming', async () => {
+        await renderComponent({
+          collapsible: true,
+          // @ts-expect-error Test fixture with partial mock
+          generatedAnswerState: {
+            isStreaming: true,
+          },
+        });
+
+        expect(renderShowButton).not.toHaveBeenCalled();
+      });
```

Comment:
This one should be outside of the describe block too (it contradicts the condition).